src/client/components/app/Navbar/index.tsx

```diff
@@ -116,6 +141,17 @@ export const Navbar = ({
   const { isMobile } = useWindowDimensions();
   const { NETWORK_SETTINGS } = getConfig();
   const walletNetwork = useAppSelector(WalletSelectors.selectWalletNetwork);
+  const walletNetworkName = useAppSelector(WalletSelectors.selectWalletNetworkName);
+
+  const { SUPPORTED_NETWORKS } = getConstants();
+
+  const SUPPORTED_NETWORK_OPTIONS = SUPPORTED_NETWORKS.concat('goerli').map((network) => {
```

Comment:
@kibagateaux I'm doing this instead of just including `goerli` in `SUPPORTED_NETWORKS` from `config/constants` because doing so causes issues with the Yearn SDK which does not support Goerli.